Makefile

```diff
@@ -35,10 +35,10 @@ setup-fern:
 
 file ?= .
 test:
-	poetry run pytest -rEf -s -vv $(file)
+	poetry run pytest -rEf -s -vv -n 4 $(file)
```

Comment:
[nit] can we add a version without parallelization (like `make test-serial` from django)?

workers are slower when you are running just 1 test, which is useful for debugging the feature branch